src/lib/types/apiResponses.ts

```diff
@@ -132,6 +132,7 @@ export interface PublicEvent {
   pointValue: number;
   requiresStaff: boolean;
   staffPointBonus: number;
+  discordEvent?: string | null;
```

Comment:
Here: let's figure out if this is string or null or undefined so we can just have one default value when its unset